docs/utilities/parser.md

```diff
@@ -2,168 +2,85 @@
 title: Parser (Pydantic)
 description: Utility
 ---
+
 <!-- markdownlint-disable MD043 -->
 
-This utility provides data parsing and deep validation using [Pydantic](https://pydantic-docs.helpmanual.io/){target="_blank" rel="nofollow"}.
+The Parser utility simplifies data parsing and validation using [Pydantic](https://pydantic-docs.helpmanual.io/){target="\_blank" rel="nofollow"}. It allows you to define data models in pure Python classes, parse and validate incoming events, and extract only the data you need.
 
 ## Key features
 
-* Defines data in pure Python classes, then parse, validate and extract only what you want
-* Built-in envelopes to unwrap, extend, and validate popular event sources payloads
-* Enforces type hints at runtime with user-friendly errors
-* Support only Pydantic v2
+- Define data models using Python classes
+- Parse and validate Lambda event payloads
+- Built-in support for common AWS event sources
+- Runtime type checking with user-friendly error messages
+- Compatible with Pydantic v2
 
 ## Getting started
 
 ### Install
 
-!!! info "This is not necessary if you're installing Powertools for AWS Lambda (Python) via [Lambda Layer/SAR](../index.md#lambda-layer){target="_blank"}"
-
-You need to bring Pydantic v2.4.0 or later as an external dependency.
-
-Add `aws-lambda-powertools[parser]` as a dependency in your preferred tool: _e.g._, _requirements.txt_, _pyproject.toml_.
-
-### Defining models
-
-You can define models to parse incoming events by inheriting from `BaseModel`.
-
-```python title="Defining an Order data model"
-from aws_lambda_powertools.utilities.parser import BaseModel
-from typing import List, Optional
-
-class OrderItem(BaseModel):
-	id: int
-	quantity: int
-	description: str
+Parser supports Pydantic v2. Each Pydantic version requires different dependencies before you can use Parser.
```

Comment:
We are supporting only Pydantic v2 now, so, the part "each pydanic" is not necessary anymore. What about: Parser supports Pydantic v2. To use Parser, you'll need to install the necessary dependencies for Pydantic v2 beforehand.?

Reply:
Great call, updated!